vms/vms/urls.py

```diff
@@ -16,4 +21,13 @@
     url(r'^registration/', include('registration.urls', namespace='registration')),
     url(r'^shift/', include('shift.urls', namespace='shift')),
     url(r'^volunteer/', include('volunteer.urls', namespace="volunteer")),
+    url(r'^portal', TemplateView.as_view(template_name='home/home.html'),name='home'),
+    url(r'^login/$',
+         anonymous_required(auth_views.login),
+        {'template_name': 'authentication/login.html'},
```

Comment:
@amruthasangeeth Okay. Cool. Then please do this change. It is more readable and closely related to the indentation others are following.